api/docs/release.dox

```diff
@@ -282,6 +282,11 @@ Further non-compatibility-affecting changes include:
  - Added proc_get_vector_length_bytes() for AArch64. This returns the current
    vector length on all ARMv8 hardware including hardware which supports the
    Scalable Vector Extension (SVE).
+ - The addition of AArch64 SVE support required changes to dr_mcontext_t and
+   dr_simd_t. In addition to the vector registers simd[] in dr_mcontext_t,
+   there are also SVE predicate registers svep[] and the SVE first-fault
+   register, ffr. The dr_simd_t register storage has increased in size from 128
```

Comment:
As in all Doxygen comments, use `#` to make links to all these types: `#dr_mcontext_t`, `#dr_simd_t`.

Reply:
Done.

Reply:
Tests are failing with the following Doxygen build error:
```
/home/github/_work/dynamorio/dynamorio/api/docs/release.dox:187: warning:
explicit link request to 'dr_simd_t' could not be resolved
```
Though no such error for `#dr_mcontext_t` which looks the same in terms of type declaration.

Reply:
Maybe Doxygen is confused by all the ifdefs (despite the preset defines in its config); maybe the underscore one _dr_simd_t is known to it; checking the html may tell.

Reply:
`dr_simd_t` doesn't appear in the current published HTML at https://dynamorio.org/search?query=dr_simd_t.

The reason Doxygen can't generate a proper HTML definition and thus link to it is because the parser is getting confused by the `ALIGN_VAR()` attribute macro in:
```
typedef union ALIGN_VAR(16) _dr_simd_t {
```
Rendering it as a function `ALIGN_VAR()` rather than a data structure:
```
union ALIGN_VAR (16) _dr_simd_t
```
See `Functions` in https://dynamorio.org/dr__defines_8h.html

Trying to link with `#_dr_simd_t` doesn't work either.